core/src/main/java/com/yahoo/oak/OrderedChunk.java

```diff
@@ -225,7 +235,7 @@ private int getLastSortedEntryIndex(int sortedCount) {
     int compareKeyAndEntryIndex(KeyBuffer tempKeyBuff, K key, int ei) {
         boolean isAllocated = entryOrderedSet.readKey(tempKeyBuff, ei);
```

Comment:
what we can do here ?
there is a situation where isAllocated will be false.

Reply:
I am referring to the assert that comes directly after this line
`assert isAllocated;` do we want to remove it?